File: python/7 kyu/258.py
# https://www.codewars.com/kata/56d5166ec87df55dbe000063/train/python

# Passed

from math import floor

def sum_average(arr):
    averages = []
    for sub_arr in arr:
        average = sum(sub_arr) / len(sub_arr)
        averages.append(average)
        
    result = floor(sum(averages))
    return result

output = sum_average([[52, 64, 84, 21, 54], [44, 87, 46, 90, 43]])
print(output)

File: nodejs/7 kyu/258.js
// https://www.codewars.com/kata/564f3d49a06556d27c000077/train/javascript

// Passed

function pattern(n) {
  let rows = [];
  for (let num = n; num >= 1; num--) {
    let digits = String(n - num + 1).split("");
    let digit = digits[digits.length - 1];

    let row = " ".repeat(n - num) + (" " + digit).repeat(num);
    rows.push(row);
  }

  let result = rows.join("\n");
  return result;
}

const output = pattern(12);
console.log(output);